### DISCUSSION TITLE
ScriptReference/MonoBehaviour.OnApplicationFocus

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#515] [Notes]
# ScriptReference/MonoBehaviour.OnApplicationFocus

https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnApplicationFocus.html

**Comment by radiatoryang** [comment]
For WebGL, it's a little complicated. See this thread: https://forum.unity.com/threads/onpause-events-for-webgl-builds.432775/
